.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -246,6 +255,11 @@ jobs:
             until docker manifest inspect quay.io/${{ github.repository_owner }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
           done
 
+      - name: Checkout code
+        uses: actions/checkout@dcd71f646680f2efd8db4afa5ad64fdcba30e748
+        with:
+          persist-credentials: false
+
```

Comment:
Oh wait, I think this was reverted by mistake? Re-adding a previous conversation:

This will check out the code from the `master` branch due to the `issue_comment` trigger.

This is sound for `master` PRs as we don't want potential attackers to modify the Helm chart and run funky stuff in our CI from a PR, however for stable branches we should point to the actual target branch instead (`v1.10` and `v1.11`), like we do for the L4LB backports: https://github.com/cilium/cilium/blob/3625f523feb04b77674a6510b6635587a5b52880/.github/workflows/tests-l4lb-v1.10.yaml#L149-L152

Reply:
Discussed offline: risk of attacker editing Helm chart is not actually a threat, but we do need to change `ref` to point to the same SHA as in `--helm-set=image.tag=${SHA}` so that Helm chart is actually the one from the PR.